assets/onboarding/query-string-router/Route.jsx

```diff
@@ -0,0 +1,28 @@
+import { useQueryStringRouter } from '../query-string-router';
+
+/**
+ * Route component. It works inner the `QueryStringRouter context.
+ *
+ * @param {Object}  props
```

Comment:
@yscik, I documented the components with JSDoc, but I can't document in that way that you sent because the lint was not passing. So I documented like this. Anything, feel free to reach me out ;)